lmcache/storage_backend/local_backend.py

```diff
@@ -562,3 +563,214 @@ def close(self):
 
     def __del__(self):
         self.close()
+
+
+@_lmcache_nvtx_annotate
+@torch.inference_mode()
+def save_disk_with_callback(
+    path: str,
+    kv_obj: KVObj,
+    key: CacheEngineKey,
+) -> Tuple[CacheEngineKey, float, int]:
+    save_file({"kv_chunk": kv_obj.data.contiguous()}, path)
+    return key, _get_size(kv_obj.data), kv_obj.chunk_idx
```

Comment:
Why this function is called `save_disk_with_callback`, where's the callback?

Reply:
```
future = self.proc_pool_executor.submit(save_disk_with_callback, path,
                                                kv_obj, key)
        future.add_done_callback(self.save_end_callback)
```
in `put_nonblocking` function